src/handler.ts

```diff
@@ -1,17 +1,56 @@
 import 'source-map-support/register';
-import { Context, APIGatewayEvent } from 'aws-lambda';
+import { ScheduledEvent, Context, Callback } from 'aws-lambda';
+import { sendModifiedTestStations } from './eventbridge/send';
+import logger from './observability/logger';
+import { DynamoTestStation } from './Interfaces/DynamoTestStation';
 
 const {
   NODE_ENV, SERVICE, AWS_PROVIDER_REGION, AWS_PROVIDER_STAGE,
 } = process.env;
 
+interface EventDetail {
+  lastModifiedDate: string;
+}
+
+const ONE_DAY = 1;
+
 console.log(
   `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_PROVIDER_STAGE}'\n region: '${AWS_PROVIDER_REGION}'\n\n`,
 );
 
-const handler = (event: APIGatewayEvent, _context: Context) => {
-  console.log('event');
-  console.log(JSON.stringify(event, null, 2));
+const handler = async (event: ScheduledEvent<EventDetail>, _context: Context, callback: Callback): Promise<void> => {
+  try {
+    logger.debug(`Function triggered with '${JSON.stringify(event)}'.`);
+
+    let lastModifiedDate: Date;
+    if (event?.detail?.lastModifiedDate) {
+      lastModifiedDate = getDateFromManualTrigger(event.detail.lastModifiedDate);
+    } else {
+      const now = new Date(Date.now());
+      lastModifiedDate = new Date(now.setDate(now.getDate() - ONE_DAY));
+    }
+
+    // TODO: replace with response from Dynamics
+    // const modifiedTestStations = await getModifiedTestStations(lastModifiedDate);
+    if (lastModifiedDate === new Date(Date.now())) { return; } // TODO: REMOVE
```

Comment:
Is this for removal now or after Aarons work has gone in?

Reply:
Yes, they're placeholders for @aaron-downing 's work (we needed to use `lastModifiedDate` to remove a compilation error)